src/Compilers/Core/Portable/Generated/Operations.Generated.cs

```diff
@@ -9054,6 +9061,11 @@ internal sealed partial class OperationCloner : OperationVisitor<object?, IOpera
         public override IOperation DefaultVisit(IOperation operation, object? argument) => throw ExceptionUtilities.Unreachable;
         private ImmutableArray<T> VisitArray<T>(ImmutableArray<T> nodes) where T : IOperation => nodes.SelectAsArray((n, @this) => @this.Visit(n), this);
         private ImmutableArray<(ISymbol, T)> VisitArray<T>(ImmutableArray<(ISymbol, T)> nodes) where T : IOperation => nodes.SelectAsArray((n, @this) => (n.Item1, @this.Visit(n.Item2)), this);
+        public override IOperation VisitBranch(IBranchOperation operation, object? argument)
+        {
+            var internalOperation = (BranchOperation)operation;
+            return new BranchOperation(internalOperation.Target, internalOperation.BranchKind, internalOperation.OwningSemanticModel, internalOperation.Syntax, internalOperation.IsImplicit);
```

Comment:
Similar comment to `IEmptyOperation`. Assume this is a general pattern now so will wait for your response before I dig into the other `VisitXXX` calls here.